### DISCUSSION TITLE
inline function 的 inline 怎麼翻？

### DISCUSSION THREAD

**Discussion by mattwang44** [python/python-docs-zh-tw#558] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

inline

### 術語翻譯

我不知道QQ

### 說明

C API 常提到的 `inline function` 不知道怎麼翻譯
- 目前翻譯為「行內函式」（如 glossary 頁面內 [lambda 條目](https://docs.python.org/zh-tw/3/glossary.html#term-lambda) 的敘述有出現）
- 繁中文章多翻譯為「內嵌函式」
- 簡中文章常翻譯為「內聯函式」

### 參考資料

MS doc 為「內嵌函式」([ref](https://learn.microsoft.com/zh-tw/cpp/cpp/inline-functions-cpp?view=msvc-170))、繁中版 C++ primer 好像也是

**Comment by rockleona** [comment]
我還是支持「行內函式」雖然好像沒有其他 Reference XD

但「內嵌」會讓我想到 embedded 這個詞，所以我認為「行內」比較直覺
